tests/helpers.bash

```diff
@@ -10,6 +10,15 @@ BUILD_NUM="${BUILD_NUMBER:-0}"
 JOB_BASE="${JOB_BASE_NAME:-local}"
 BUILD_ID="${JOB_BASE}-${BUILD_NUM}"
 
+# Prefer local build if binary file detected.
```

Comment:
In general I agree with @aanm that the `cilium-agent` and `cilium-docker` would not need this but it's correct to have a proper `PATH` for all binaries and it clearly documents the need to account for the `PATH` having to point to the build location if someone wants to run outside of a systemd environment. I don't see any downsides of having this, it's clearly an improvement.